docker-compose.yml

```diff
@@ -160,3 +160,21 @@ services:
     depends_on:
       - handshake
       - handshake-api
+
+  synckeys:
+    build:
+      context: docker/synckeys
+      dockerfile: Dockerfile
+    container_name: synckeys
+    restart: on-failure
+    logging: *default-logging
+    volumes:
+      - ./docker/data/keys:/keys
```

Comment:
please call the container data directory the same name as the container name (so `synckeys` instead of `keys`) to keep consistency

Reply:
Sure, good point.